### ISSUE TITLE
The issue with Network chosing section of VCH deployment is still unresolved

### ISSUE THREAD

**Issue by daydrim** [vmware/vic-ui#631]
**VIC-UI version:**
v1.4.4.1405

**Platform details:**
Client: Windows 10
VC: 6.7.0 Build 10244857

**Browser details:**

IE 11 - UI won't work at all
Chrome 70.0.3538.102 (Official Build) (64-bit) OK
Firefox 63.0.1 (64-bit) OK

**Actual behavior:**
The issue with Network section of VCH deployment is still unresolved. No networks can be chosen.
This issue is described in the following article: 
[](https://kb.vmware.com/s/article/56471)
Resolution:
This issue occurs when ESXI Cluster is nested within folders.
This is a known issue affecting vSphere Integrated Containers if deployed in Nested Environments. There is currently no resolution. This issue is to be addressed in future versions of the product. 

The same applied to Nesting of Distributed Switch. 

**Expected behavior:**
Expected behavior - all the networks can be chosen regardless of the folders and nesting 

Is this behavior planned to resolving in new versions?

**Comment by renmaosheng** [comment]
Meina,
     could you please take a look? thanks.

**Comment by zhoumeina** [comment]
@daydrim ， could you please make a screenshot of your dc structure and the network hanging screenshot?Or I can log in to your vcenter server?I can take a look.

**Comment by zhoumeina** [comment]
@daydrim , nested folder clusters is not supported in 1.4.4. I tested the master branch with IE11. It is totally ok with it. Because vic 1.4.4 already released, you can use vic 1.5 instead. It will released on December.

**Comment by daydrim** [comment]
@zhoumeina , Hello, it happened in Chrome and Firefox.
I can't provide the screenshot, but I can make an example of structure in text below: Network Tab
```
->VC
-->DC1
--->Folder1
---->Folder2
----->dVS
```

If we move dVS to the VC level it works fine.
Like this:
`
->VC
-->DC1
--->vDS
`

PS. I've seen the improvement in the code [create-vch-wizard.service.ts](https://github.com/vmware/vic-ui/commit/c52bf7fc120060d4bc54e8d5ac07b977a9452f3a#diff-f361ad37b31a58cab387befb69ffe05a)

I hope it helps ) so this improvement can be tested in December as far I understand )

**Comment by zhoumeina** [comment]
@daydrim , exactly. I reproduced your problem in my environment. Yes. vic 1.4 does not support nested folders. This can be fixed in 1.5
